gd32a50x/cmsis/gd/gd32a50x/source/system_gd32a50x.c

```diff
@@ -809,7 +809,7 @@ void SystemCoreClockUpdate(void)
         /* PLL multiplication factor */
         pllmf = GET_BITS(RCU_CFG0, 18, 21);
         pllmf += ((RCU_CFG0 & RCU_CFG0_PLLMF_4) ? 15U : 0U);
-        pllmf += ((0xFU == (RCU_CFG0 & RCU_CFG0_PLLMF)) ? 1U : 2U);
+        pllmf += ((RCU_CFG0_PLLMF == (RCU_CFG0 & RCU_CFG0_PLLMF)) ? 1U : 2U);
```

Comment:
Like gd32l23x, add this exception to https://github.com/zephyrproject-rtos/hal_gigadevice#exceptions.